assets/css/footer.css

```diff
@@ -1,5 +1,5 @@
 .footer-text {
-  background-color: var(--primary-color);
+  background-color: black;
```

Comment:
Please use variable here. Declare black in your vars.

Reply:
Done.